bootcamp/contrib/student_15.py

```diff
@@ -0,0 +1,12 @@
+def CaseConversion(string):
+    """
+    Change Upper Case to Lower Case; Vise versa
+    """
+    CaseDict = {
```

Comment:
Variable names should be lowercase, see [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names).